source/index-management.rst

```diff
@@ -405,20 +466,35 @@ Index View API
      * @note Drivers MAY opt to implement this method signature, the signature that
      *   takes an IndexModel as a parameter, or for those languages with method
      *   overloading MAY decide to implement both.
+     *
+     * @note Drivers MAY combine the two options types into a single one. If the options are
+     *   explicitly typed, the combined options type MUST be named DropOneIndexOptions or an acceptable
+     *   variation.
      */
-    dropOne(keys: Document, options: IndexOptions): Result;
+    dropOne(keys: Document, indexOptions: IndexOptions, options: Optional<DropOneIndexOptions>): Result;
 
     /**
      * @see Comments above.
      */
-    dropOne(model: IndexModel): Result;
+    dropOne(model: IndexModel, options: Optional<DropOneIndexOptions>): Result;
 
     /**
      * Drops all indexes in the collection.
      */
     dropAll(): Result;
   }
 
+  interface CreateOneIndexOptions {
```

Comment:
I'd say we can reuse the common options type here as well

Reply:
See discussion above about this